pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -53,17 +56,32 @@
  * If the input is single value, the output type will be input type. Otherwise, the output type will be double.
  */
 public class AggregateOperator extends BaseOperator<TransferableBlock> {
+
+  interface Merger extends BiFunction<Object, Object, Object> {
+  }
+
   private static final String EXPLAIN_NAME = "AGGREGATE_OPERATOR";
+  private static final Map<String, Merger> MERGERS = ImmutableMap
```

Comment:
this private static member can be in Accumulators?

Reply:
IMO it's nice to have constants at the top of the file, but don't feel strongly. I'll move it